nuxeo-java-client/src/main/java/org/nuxeo/client/NuxeoClient.java

```diff
@@ -242,8 +213,39 @@ public boolean isCacheEnabled() {
         return nuxeoCache != null;
     }
 
-    /** Services **/
+    public NuxeoClient refreshCache() {
+        if (isCacheEnabled()) {
+            nuxeoCache.invalidateAll();
+        }
+        return this;
+    }
+
+    /******************************
+     * Services *
+     ******************************/
+
+    /**
+     * @return The Nuxeo server version.
+     */
+    public NuxeoVersion getServerVersion() {
+        if (serverVersion == null) {
+            try {
+                // Remove API_PATH from the base url
+                // Get repository capabilities on CMIS
+                Response response = get(retrofit.baseUrl().toString().replaceFirst(ConstantsV1.API_PATH, "") + "/json/cmis");
+                String body = response.body().string();
+                String version = body.replaceFirst(".*\"productVersion\":\"([^\"]*)\".*", "$1");
```

Comment:
Why not just use a matcher and extract the match? Or even properly parse the JSON?